### ISSUE TITLE
Crashing when opening folders, crashing when searching, crashing when viewing records

### ISSUE THREAD

**Issue by StanoRiga** [MiniKeePass/MiniKeePass#546]
Hi all,
I`m upset. I loved this app, but since iOS11 on my iPphone 7+ it keeps on crashing in various situations.
It is nearly impossible to get a password out of it.
Please fix that. 
If I can support with logs or testing, please let me know.

**Comment by Erreu** [comment]
Are you running 1.7.0 or 1.7.1?

**Comment by tssmith7** [comment]
The most useful thing would be an example database that crashes without any personal information in it.  See issue #526.

**Comment by StanoRiga** [comment]
Sorry, your right. I should have been more precise.
I'm running 1.71. with iOS 11.0.1

The crashes happen if
a) I enter the first character in the search bar at root.
b) I klick on a folder with 185 entries. All other folders with less entries do not crash.

In my titel I wrote about crashing when viewing records. I cannot reproduce it, maybe it was just my nerves.
I tried to reproduce behavior b) and created a fresh database with 256 records in a sub-folder. Everything is finde. I am rather at a loss.
I use the database on android and windows pc as well, everything is fine. only iPhone App does have the malfunction.
